scripts/tech-debt-burndown-report.ts

```diff
@@ -115,6 +115,47 @@ const fileErrorCounts = countArrayEntries(prData.map((error) => error.value.path
 
 const errorCodeCounts = countArrayEntries(prData.map((error) => error.value.tsError.value.errorString));
 
+//// unused exports check ////
+// TODO(STENCIL-454): Remove or change up this report once we've eliminated unused exports
+
+/**
+ * ts-prune includes this string on lines in its output corresponding to items
+ * which are exported and not imported anywhere but which which *are* used in their
+ * home modules. We want to exclude these and get only the truly dead code.
+ */
+const USED_IN_MODULE = '(used in module)';
+
+/**
+ * Load in data from the unused exports reports
+ */
+const unusedExportsMain = String(fs.readFileSync('./unused-exports-main.txt'));
+const unusedExportsPR = String(fs.readFileSync('./unused-exports-pr.txt'));
+
+function processUnusedExports(rawFileContents: string) {
```

Comment:
Minor - can we JSDoc this function, and declare its return type in the function sig?